### ISSUE TITLE
[FEAT] eva02 모델로 데이터 학습시키고 추론 뽑음

### ISSUE THREAD

**Issue by limhongcheol** [beom-jo-city/catch_mind#2]
## Todo
- [ ] 데이터셋 가져오기
- [ ] eva02모델로 넣고 돌리기

## Further Know
- batch_size = 256
- loss = focal_loss
- scheduler = cosine_annealing
- early stop = True
